src/components/pages/contact/contact-form.tsx

```diff
@@ -28,6 +30,26 @@ interface FormData {
   message: string;
 }
 
+/**
+ * This is a honeypot field but we want to disguise it including the wording
+ * and hope the bots can't filter them by any heuristic
+ * (hopefully ignoring this comment
+ **/
+
+const MagicFieldContainer = styled.div`
+  ${hideVisually()}
+`;
+const MagicField = ({ label, control }) => {
```

Comment:
Good idea 👍 I am curious whether this spam is intercepted

Reply:
Yeah some logging would help as suggested by Fabian. Let's see.